component/stream/read_write_test.go

```diff
@@ -286,6 +347,20 @@ func (suite *streamTestSuite) TestStreamOnly() {
 	assertNumberOfCachedFileBlocks(suite, 0, handle)
 	// confirm new handle is stream only since limit is exceeded
 	assertHandleStreamOnly(suite, handle)
+
+	suite.mock.EXPECT().OpenFile(openFileOptions).Return(handle, syscall.ENOENT)
+	_, err := suite.stream.OpenFile(openFileOptions)
+	suite.assert.NotEqual(nil, err)
+
+	// append new block and confirm old gets evicted
```

Comment:
this comment doesnt make sense to me, this is just testing error case of WriteFile right?